spec/index.html

```diff
@@ -0,0 +1,546 @@
+
+    <p its-locale-filter-list="zh-hans" lang="zh-hans">用户可以通过 Widget 获取正在播放的音乐信息并进行播放控制。</p>
+    <p its-locale-filter-list="en" lang="en"> Users can get the information of playing musicand and control the playback through a Miniapp widget.</p>
+
+
+    <p its-locale-filter-list="zh-hans" lang="zh-hans">用户在浏览视频或者图片类 MiniApp 时，自动识别内容中的商品。当用户点击这个商品时，打开电商 MiniApp 相关的 MiniApp Widget。
+      这个 Widget 可以浮窗形式出现，呈现商品的一些基本信息。用户可以点击Widget 上的详情按钮，跳转到电商 MiniApp 的页面来购买。</p>
+    <p its-locale-filter-list="en" lang="en">In video or picture MiniApps, the products
+      in the content can be identified automatically. When user clicks on this item,
+      the MiniApp Widget related to the e-commerce MiniApp is opened. This Widget can
+      appear as a floating window, presenting some basic information about the
+      product. Users can click the details button on the Widget to jump to the
+      e-commerce MiniApp page to purchase.</p>
+
+    <p its-locale-filter-list="zh-hans" lang="zh-hans">用户询问语音助手时，例如询问汇率，语音助手打开汇率相关的 MiniApp Widget，呈现相关信息。</p>
+    <p its-locale-filter-list="en" lang="en">When voice assistant is asked about
+    exchange rate, for instance, voice assistant can open related MiniApp Widget to
+    present relevant information.</p>
+  </section>  
+
+  <section id='addressing'>
+    <h2>
+      <span its-locale-filter-list="en" lang="en">Addressing</span>
+      <span its-locale-filter-list="zh-hans" lang="zh-hans">Addressing</span>
+    </h2>
+
+    <p its-locale-filter-list="zh-hans" lang="zh-hans">与 MiniApp page 类似，定位 MiniApp widget 及 Widget 中的资源的过程符合 <a
+    href="https://w3c.github.io/miniapp-addressing/">Miniapp Addressing</a> 定义的标准。</p>
+    <p its-locale-filter-list="en" lang="en">Similar to MiniApp page, the process of locating widget and the resources in it complies with the specification defined
+      by <a href="https://w3c.github.io/miniapp-addressing/">Miniapp Addressing</a>.</p>
+
+    <p its-locale-filter-list="zh-hans" lang="zh-hans">与 page 不同的是， MiniApp widget 并不存在 start widget。因此，指定某一个 MiniApp widget 需要显式说明 Widget 的 path，如下例所示：</p>
+    <p its-locale-filter-list="en" lang="en">Unlike Miniapp page, there is no start widget for a MiniApp widget. Therefore, specifying a MiniApp widget needs to
+    specify the path of the widget explicitly, as shown in Example:</p>
+
+    <pre class="example" title="URI example for a MiniApp widget|MiniApp Widget的URI示例">
+      platform://miniapp/foo/widgets/index
+      platform://miniapp/foo/widgets/index?category=img
+    </pre>
+  </section>
+
+  <section id='packaging'>
+    <h2>
+      <span its-locale-filter-list="en" lang="en">Packaging</span>
+      <span its-locale-filter-list="zh-hans" lang="zh-hans">Packaging</span>
+    </h2>
+
+    <p its-locale-filter-list="zh-hans" lang="zh-hans">包含 Widget 的 MiniApp 包结构符合如下要求：</p>
+    <p its-locale-filter-list="en" lang="en">The structure of MiniApp package containing a Widget should meets the following requirements:</p>
```

Comment:
I suggest using a list for the requirements below, which is a bit clearer than `p`. See [clreq](https://github.com/w3c/clreq/blob/2b8f9ccc80c7731bb1662e05d1aea086a9c82b4b/index.html#L222-L268) for example markup.